r/dm06_swap_parent_child_info.R

```diff
@@ -26,8 +26,8 @@ print(paste0("Opened: ", input_name))
 
 
 map_sample_type <- c(
```

Comment:
I ran command+i and this happened automatically - my Rstudio is also set to remove whitespace under some circumstances

Reply:
This can be ignored